policies/gitlab/repository.rego

```diff
@@ -84,16 +84,16 @@ missing_default_branch_protection {
 #   remediationSteps: [Make sure you have owner permissions, Go to the projects's settings -> Repository page, Enter "Protected branches" tab, select the default branch. Set the allowed to merge to "maintainers" and the allowed to push to "No one"]
 #   severity: MEDIUM
 #   threat: Rewriting project history can make it difficult to trace back when bugs or security issues were introduced, making them more difficult to remediate.
-default missing_default_branch_protection_force_push = false
+default missing_default_branch_protection_force_push = true
 
-missing_default_branch_protection_force_push {
+missing_default_branch_protection_force_push = false {
```

Comment:
This part of the rule causes opa to return the "complete rules must not produce multiple outputs" error, because the same rule returns three values: true, false, and the default.

You need to remove it and modify the last part of the rule to check whether:
1. The default branch is protected, and
2. The default branch doesn't allow for force push

For example: 

```
missing_default_branch_protection_force_push = false {
	default_protected_branches := [protected_branch | protected_branch := input.protected_branches[_]; protected_branch.name == input.default_branch]
	count(default_protected_branches) > 0
	rules_allow_force_push := [rule_allow_force_push | rule_allow_force_push := default_protected_branches[_]; rule_allow_force_push.allow_force_push == true]
	count(rules_allow_force_push) == 0
}
```

Reply:
Hi @noamd-legit,
I have updated the PR. 
Thanks for the guidance.